recipes/install.rb

```diff
@@ -48,3 +47,11 @@
     cwd     "#{node[:valhalla][:src_dir]}/#{repo}"
   end
 end
+
+# link the lua transforms from the checkout
+%w(vertices edges admins).each do |lua|
```

Comment:
since this has to do with actually cutting tiles maybe its better in the scripts as well?